internal/gapicgen/cmd/genbot/github.go

```diff
@@ -27,7 +27,7 @@ import (
 	"time"
 
 	"cloud.google.com/go/internal/gapicgen/generator"
-	"github.com/google/go-github/v32/github"
+	"github.com/google/go-github/v33/github"
```

Comment:
Please run `go mod tidy` from this dir. We need to make sure `go.mod` & `go.sum` get upgraded.

Reply:
Done. PTAL